packages/bpk-component-segmented-control/src/BpkSegmentedControl.module.scss

```diff
@@ -37,9 +38,10 @@
 .bpk-segmented-control {
   min-height: tokens.bpk-spacing-xl();
   padding: tokens.bpk-spacing-md() tokens.bpk-spacing-base();
-  flex: 1;
   border: none;
+  text-align: center;
```

Comment:
it seems user agent stylesheet has default set text-align to center, do we need rewrite here?

![image](https://github.com/user-attachments/assets/f9de9224-fc8e-4dda-9149-c5d7888a9e3a)